src/screens/RequestSwap/SelectSwapOption.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from 'react';
+import { View, StyleSheet, TouchableOpacity } from 'react-native';
+import { Layout, Text, Icon, Button } from '@ui-kitten/components';
+
+interface SelectSwapOptionProps {
+    navigation;
+    route;
+}
+
+export const SelectSwapOption = (props: SelectSwapOptionProps) => {
+    const [selectedOption, setSelectedOption] = useState<number>(0);
```

Comment:
is there any way we can we add index and make this look for index number? usually a function will look for an option's index

Reply:
this part isn't completed yet, I need an api to query the database for available people. I will revisit this portion once the API is complete

Reply:
ok